### PR TITLE
Issue #1489: Add generic window resize handler and apply it to JS in core

### PR DESCRIPTION
**Pull request by wesruv**

### CODE REVIEW

File: core/misc/tableresponsive.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ function TableResponsive (table) {
 
   // Attach a resize handler to the window.
   $(window)
+    // @todo - I have no idea what this is doing.
```
Comment (wesruv) [inline core/misc/tableresponsive.js]: I'm not sure what's happening here and I'm not sure how to rewrite it...

**Comment by wesruv** [comment]
I think this isn't firing something on admin menu properly. My guess is I need to call a resize function on page load and then the resize handler will pick up the rest.

**Comment by wesruv** [comment]
@quicksketch ^^

**Comment by wesruv** [comment]
All fixed

**Comment by wesruv** [comment]
@jenlampton @quicksketch this is blocking some other work that I'm doing, anyway we can get this reviewed?

**Comment by quicksketch** [comment]
This looks really good. We haven't yet branched 1.4.x (though we should shortly). This should probably be a 1.5.x feature (so merging into 1.x after we make the 1.4.x branch).

**Comment by quicksketch** [comment]
Sweet, I hadn't seen this was ready again since 1.4 came out. I'll work on getting this merged, very excited for this feature!

**Comment by quicksketch** [comment]
I've manually merged this in 78d0f38.

**Comment by quicksketch** [comment]
As a new feature, I don't think this should be in 1.4.x, but this will be in 1.5.0. Thanks @wesruv!

**Pull request closed without merging**